pyc_object.h

```diff
@@ -31,6 +31,8 @@ class PycRef {
             m_obj->delRef();
     }
 
+    // Most operators should deal with m_unpack but we leave it as is for now
```

Comment:
The copy and move assignment operators should definitely handle it, to avoid surprises later...